src/util/user-home-dir.js

```diff
@@ -4,7 +4,11 @@ import ROOT_USER from './root-user.js';
 
 const path = require('path');
 
+const isNotFakeRoot = typeof process.env.FAKEROOTKEY === 'undefined';
```

Comment:
Agree with the concern but looks like `ROOT_USER` is only used here and with the assumption of root permissions so probably it's a good idea to put it there. @bestander @arcanis thoguhts?